kiwi/bootloader/config/zipl.py

```diff
@@ -134,7 +113,7 @@ def setup_disk_image_config(
         :param string hypervisor: unused
         :param string kernel: kernel name
         :param string initrd: initrd name
-        :param string boot_options: kernel options as string
+        :param string boot_options: unused
```

Comment:
I'd say this needs to be `dict` or `dictionary` instead of string.

Reply:
yes, this was also wrong in the grub code, also fixed it there